src/coreclr/tests/helixpublishwitharcade.proj

```diff
@@ -82,6 +82,7 @@
     <HelixRuntimeRid Condition="'$(TargetOSSpec)' == 'Linux'">linux-$(TargetArchitecture)</HelixRuntimeRid>
     <HelixRuntimeRid Condition="'$(TargetOSSpec)' == 'Linux_musl'">linux-musl-$(TargetArchitecture)</HelixRuntimeRid>
     <HelixRuntimeRid Condition="'$(TargetArchitecture)' == 'wasm'">browser-wasm</HelixRuntimeRid>
+    <HelixRuntimeRid Condition="'$(TargetOS)' == 'Android'">android-x64</HelixRuntimeRid>
```

Comment:
It looks like `TargetOSSubgroup` is empty, could we also use `TargetOSSpec` here?
`<HelixRuntimeRid Condition="'$(TargetOSSpec)' == 'Android'">android-$(TargetArchitecture)</HelixRuntimeRid>`

I'm wondering if we can change 
`<HelixRuntimeRid Condition="'$(TargetArchitecture)' == 'wasm'">browser-wasm</HelixRuntimeRid>`
to be 
`<HelixRuntimeRid Condition="'$(TargetOSSpec)' == 'Browser'">browser-$(TargetArchitecture)</HelixRuntimeRid>`

Reply:
Fixed, thanks!

Reply:
Not sure if we plan to support other arch for brewer, other than wasm. So I didn't touch the logic for wasm.